lib/index.js

```diff
@@ -18,6 +18,8 @@ const DEFAULT_OPTIONS = {
 
 const SPLIT_CHARS = '[^a-zA-Z_]'
 
+const DEFAULT_KEYWORD = 'default'
```

Comment:
Do you think 'whitespace' would make more sense? As I assume this is only what this will be used for?

Reply:
naming ist not the best but `whitespace` would be wrong. For example query `(f2)`. would result in default segment `f1`. Do you have an idea how to name it?

Reply:
Aah I see. In that case, no, default is sufficient :)